constellations/orion/src/controllers/transaction_controller.go

```diff
@@ -0,0 +1,86 @@
+package controllers
+
+import (
+	"net/http"
+
+	"github.com/ahsu1230/mathnavigatorSite/constellations/orion/src/domains"
+	"github.com/ahsu1230/mathnavigatorSite/constellations/orion/src/repos"
+	"github.com/gin-gonic/gin"
+)
+
+func GetAllTransactions(c *gin.Context) {
```

Comment:
Can I get one more endpoint here? It's a pretty simple one: At GET `api/transactions/types`, can the user get a list of all types of PAYMENT_TYPES? So it should simply return a list of strings: `[pay_paypal, pay_charge, pay_cash, ...]`

This is so the front-end and back-end will always be in sync with what transaction types there can be. You will only need a controller_test for this, no need for an integration test since there's no repo stuff involved here.